### DISCUSSION TITLE
Replace cd with z in Powershell

### DISCUSSION THREAD

**Discussion by AzureHound** [ajeetdsouza/zoxide#960] [Q&A] [answered]
I’m using PowerShell 7 on Windows 11, and I’d like to replace the default cd command with z (from [Zoxide](https://github.com/ajeetdsouza/zoxide)) so that whenever I type cd, it behaves like z.

I tried a couple of approaches, but I’m running into issues. Specifically:

Setting an alias: I tried Set-Alias cd z, but the built-in cd still interferes, and it doesn’t give me the desired behavior.
Using Invoke-Expression: I also tried adding --cmd cd with Invoke-Expression like this:
Invoke-Expression (& { (zoxide init --cmd cd powershell | Out-String) })
but that didn’t seem to work either.
Has anyone successfully overridden or aliased cd to z in PowerShell? Or is there a better way to make z the default navigation command instead of cd? Any suggestions or guidance would be greatly appreciated!

**Comment by ZovutVanya** [comment, marked as answer]
Strange, 
```powershell
Invoke-Expression (& { (zoxide init --cmd cd powershell | Out-String) })
```
works for me, did you place it in the right place?

**Comment by AzureHound** [comment]
Works perfectly on Linux had no issue for the last 6 months at all

```zoxide init --cmd cd fish | source```

also thanks for replying
